scripts/create_pr.sh

```diff
@@ -13,30 +13,30 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-./vuln-list-k8s -vuln-list-dir "$VULN_LIST_DIR"
+./cve-feed-osv
 
-cd "$VULN_LIST_DIR" || exit 1
+cd "cve-feed-osv" || exit 1
```

Comment:
Do we need to change the current
directory? vuln-list-k8s used to be another repository, but it's now in the same repository.